test/e2e/sagemaker/tests/_fixtures.py

```diff
@@ -0,0 +1,152 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit... there's not really a reason to prefix these module names with `_`...

Reply:
I took it from my previous experience working with the Google style guide (https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations). But I don't really know how much these apply considering we don't export any of the test methods or classes anyway.